src/main/io/osd.c

```diff
@@ -1058,6 +1051,8 @@ static void osdRefresh(timeUs_t currentTimeUs)
         armState = ARMING_FLAG(ARMED);
     }
 
+    statRssi = rssi * 100 / 1024;
```

Comment:
`scaleRange(rssi, 0, 1024, 0, 100);` is easier to understand (but also more expensive)